src/client/automation/playback/click/index.js

```diff
@@ -284,7 +280,7 @@ export default class ClickAutomation {
         var moveArguments = this._getMoveArguments();
 
         return this._move(moveArguments)
-            .then(() => this._mousedown())
+            .then(() => Promise.all([delay(ACTION_STEP_DELAY), this._mousedown()]))
```

Comment:
For the case when a mousedown event handler executing takes much time.
Imagine that mousedown handler works 50ms synchronously. In this case:

When you click on the page the delay between mousedown and mouseup action doens't depend on mousedown handlers. The timeline looks like:
0ms --------------------------50ms--------- 80ms
mousedown - --handler execution ---------mouse up

But TestCafe does this in the following way:
0ms -----------------------50ms----------------------------------130ms
mousedown - handler execution -(start 80 ms timeout) --------mouse up

As I result the delay between mousedown and mouseup is 130ms instead of 80ms. It seems we have a problem with an ASP test because of this. We are investigating the problem now